config/providers.js

```diff
@@ -3,6 +3,11 @@
 
 const deepFreeze = require('deep-freeze');
 
+const {
+  GITHUB_ORGS_OAUTH_ACCESS_SETTINGS_URL,
```

Comment:
is it github specific?

Reply:
Yes, this is how we pass the URL to Github access settings URL for a Github App.